geoips/plugins/modules/procflows/config_based.py

```diff
@@ -1691,13 +1690,18 @@ def call(fnames, command_line_args=None):
     failed_compares = {}
     for cpath in final_products:
         if cpath != "no_comparison":
-            curr_compare_outputs = find_entry_point(
-                "output_comparisons", final_products[cpath]["compare_outputs_module"]
-            )
-            curr_retval = curr_compare_outputs(cpath, final_products[cpath]["files"])
-            retval += curr_retval
-            if curr_retval != 0:
-                failed_compares[cpath] = curr_retval
+            from geoips.interfaces.module_based.output_checkers import output_checkers
+
+            for output_product in final_products[cpath]["files"]:
+                output_checker = output_checkers.get_plugin(output_product)
+                curr_retval = output_checker(
+                    output_checker,
+                    cpath,
+                    [output_product],
+                )
+                retval += curr_retval
+                if curr_retval != 0:
+                    failed_compares[cpath] = curr_retval
```

Comment:
Ah, so your solution to not having access to the plugin itself within the plugin is to just pass in the plugin explicitly ?  When we decide how to handle this natively within all plugin classes we could presumably remove the "output_checker" argument?